src/middlewared/middlewared/etc_files/local/nginx/nginx.conf.mako

```diff
@@ -187,6 +176,19 @@ http {
             rewrite ^.* $scheme://$http_host/ui/ redirect;
         }
 
+% if middleware.call_sync('vm.generate_nginx_route'):
```

Comment:
Why have this conditional? Could have this always enabled and then not need to reload nginx?
I dont think there is any harm